System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryInfoTests.cs

```diff
@@ -254,6 +254,36 @@ public void MockDirectoryInfo_FullName_ShouldReturnNormalizedPath(string directo
             Assert.AreEqual(expectedFullName, actualFullName);
         }
 
+        [TestCase(@"c:\temp\folder  ", @"c:\temp\folder")]
+        [WindowsOnly(WindowsSpecifics.Drives)]
+        public void MockDirectoryInfo_FullName_ShouldReturnPathWithTrimmedTrailingSpaces(string directoryPath, string expectedFullName)
+        {
+            // Arrange
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>());
```

Comment:
FYI,

    var fileSystem = new MockFileSystem();

It also fine? (less = more)

2x

Reply:
Thanks. Done!